### DISCUSSION TITLE
Cannot transfer domain from godaddy to vercel

### DISCUSSION THREAD

**Discussion by hsol** [vercel/community#6528] [Help]
### Summary

During loading, after payment is made, an error occurs in the final step and is refunded.

the message is:
"The provided auth code does not match with the one expected by the current registar"

### Steps to Reproduce

1. I make my domain transferable at godaddy.
2. Receive an authorization code.
3. Enter the domain I want to transfer in vercel.
4. Copy and paste the approval code into vercel.
5. Click 'Transfer In' button and it is loading.
6. Is purchased.
7. Error! "The provided auth code does not match with the one expected by the current registar"
8. Is refunded.

**Comment by versecafe** [comment]
Not sure what causes the issue but as a temporary fix you could always swap the name servers over to vercel on
```
ns1.vercel-dns.com
ns2.vercel-dns.com
```
This won't transfer the domain itself but will give you full access to it within Vercel just make sure to add it to your desired organizations Domains first

**Reply by amyegan** [reply]
This is a good option for many people. You don't need to transfer the domain to use it with a Vercel project. Other options and instructions are available at [vercel.com/docs/projects/domains/add-a-domain](https://vercel.com/docs/projects/domains/add-a-domain#)

I'm not 100% sure what is causing the auth code error based on the info here. If it's a newly purchased domain, it would be subject to a 60-day _Change of Registrant_ lock. That would cause a problem with the transfer.

https://www.icann.org/resources/pages/name-holder-faqs-2017-10-10-en

**Reply by hsol** [reply]
I had already set up DNS and was using it.
I like the Vercel ecosystem and am planning to transfer the domain as well.
Already 60 days passed, and the process for domain transfer suggested by GoDaddy has been completed.

It's not a name server issue. Exactly, I want to 'transfer in' domain to Vercel and have a collaborative relationship with Vercel ecosystem.

**Reply by amyegan** [reply]
If you let me know the domain, I can check for more info

**Reply by hsol** [reply]
@amyegan domain is "proofer.tech"
![image](https://github.com/vercel/community/assets/1524891/eda7cfba-2143-4228-8c8a-4b567ecfeb33)
![image](https://github.com/vercel/community/assets/1524891/d852b7ba-4aee-4c32-9b7c-0242f579c49f)

Thank you for not forgetting to contact me with my inquiry.

**Comment by hsol** [comment]
I changed to bug report
